bylaws/3-cf-council.md

```diff
@@ -99,7 +99,7 @@ Efficient participation and decision-making are essential for geographically dis
 
 ### Decision-Making Process
 
-- **Consensus-Based Approach:** The CFC strives to make decisions based on consensus, adhering to the Lazy Consensus model outlined in [General Decision Making][consensus]. This approach emphasizes collective agreement and accommodates diverse viewpoints.
```

Comment:
This section is currently being re-worked as part of [this PR](https://github.com/commonhaus/foundation-draft/pull/48).

Reply:
Fixing the typos is fine, I can rebase.